### PR TITLE
Add map drawer

### PR DESCRIPTION
**Pull request by KevinTss**
# Context

Add a drawer on map

Screenshot 2020-06-13 at 19 41 36

## Result

Screenshot 2020-06-13 at 19 52 38
Screenshot 2020-06-13 at 19 52 46

## Specifications

- Creation of reusable components: Badge, Text

## TODO

- [x] Add a script to move mobile drawer from a minimum height to the max height on touch
- [x] Simulate the data in the drawer parent to use it when props is passed
- [x]  Add animation to make it appear/disappear
- [ ] Check on different browser for compatibility

### CODE REVIEW

**Comment by KevinTss** [comment]
@MichielLeyman You can already check if you agree to my approach of SCSS flow. With theme variables and mixins.
I will dot the rest asap. Feel free to add topics that I forgot

File: src/components/UI/Text.svelte [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    line-height: var(--text-line-height-l);
+  }
+
+  .text.thin {
+    font-weight: normal;
+  }
+  .text.bold {
+    font-weight: bold;
+  }
+</style>
```
Comment (MichielLeyman) [inline src/components/UI/Text.svelte:51]: While I understand your thoughts behind this component, I don't know if it's necessary given global title and paragraph styles. Any modifications necessary to those styles are component-level anyways

Comment (KevinTss) [reply src/components/UI/Text.svelte:51]: I choose to do it to lock a bit the style rules into a component to make easier to scale. Inspired by [evergreen](https://evergreen.segment.com/components/typography) it really help to keep same behaviour everywhere. But we can do classes instead.

File: public/global.css [hunk 2]
```diff
@@ -20,6 +20,19 @@
...
+  --spacer-xl: 3rem;
+  --spacer-l: 2rem;
+  --spacer-m: 1rem;
+  --spacer-s: 0.8rem;
+  --spacer-xs: 0.5rem;
+  --text-font-size-l: 18px;
+  --text-font-size-m: 14px;
+  --text-line-height-l: 160%;
+  --text-line-height-m: 140%;
+  --mobile-nav-height: 9rem;
```
[the remark this answers is not in the document]
Comment (KevinTss) [reply public/global.css:35]: Not sure to get it. Can you give an example?

**Pull request merged**